src/app/ClusterStateCache.cpp

```diff
@@ -100,58 +145,44 @@ void AttributeCache::OnReportEnd()
     mCallback.OnReportEnd();
 }
 
-void AttributeCache::OnAttributeData(const ConcreteDataAttributePath & aPath, TLV::TLVReader * apData, const StatusIB & aStatus)
+CHIP_ERROR ClusterStateCache::Get(const ConcreteAttributePath & path, TLV::TLVReader & reader)
 {
-    //
```

Comment:
This has been moved to a different section in the file to better group methods together. There is no chance to this function.

Reply:
Fwiw, local git diff with `--diff-algorithm=patience --color-moved` handles this pretty well.